### DISCUSSION TITLE
PDF library with Android x support

### DISCUSSION THREAD

**Discussion by nileshamb** [afreakyelf/Pdf-Viewer#95] [Q&A]
Hello,

I am searching for a PDF library for android which supports android x, currently using [barteksc](https://github.com/barteksc)
/
[AndroidPdfViewer](https://github.com/barteksc/AndroidPdfViewer), however it is not supporting android x and therefore we are unable to disable jettifier, I will appreciate if you can confirm this or any other library is supporting android x?

thank you

**Comment by afreakyelf** [comment]
Hi, this library supports Android x and is up to date with latest android veersions. You can give it a try, please follow the instructions in https://github.com/afreakyelf/Pdf-Viewer/blob/master/README.md
